### PR TITLE
REMOVE: Smoothie Chef

### PR DESCRIPTION
**Pull request by nickswalker**
## Description

Towards #685, we agreed to cut this task during the last meeting #694

### CODE REVIEW

File: Rulebook.tex [hunk 1]
```diff
@@ -70,7 +70,6 @@ \section*{Housekeeper tasks}
 
 \section*{Party Host tasks}
 \testtocentry{test:carry-my-luggage}
-\testtocentry{test:farewell}
```
Comment (kyordhel) [inline Rulebook.tex]: Not Smoothie-Chef

File: Rulebook.tex [hunk 2]
```diff
@@ -103,14 +102,10 @@ \chapter{Tests in Stage II}
 \section*{Housekeeper tasks}
 \testtocentry{test:clean-the-table}
 \testtocentry{test:egpsr}
-\testtocentry{test:find-my-disk}
```
Comment (kyordhel) [inline Rulebook.tex]: Not Smoothie-Chef

File: Rulebook.tex [hunk 3]
```diff
@@ -103,14 +102,10 @@ \chapter{Tests in Stage II}
...
-\testtocentry{test:set-the-table}
```
Comment (kyordhel) [inline Rulebook.tex]: Not Smoothie-Chef

File: Rulebook.tex [hunk 4]
```diff
@@ -103,14 +102,10 @@ \chapter{Tests in Stage II}
...
 
 \section*{Party Host tasks}
 \testtocentry{test:hand-me-that}
-\testtocentry{test:stickler-for-the-rules}
```
Comment (kyordhel) [inline Rulebook.tex]: Not Smoothie-Chef

**Comment by nickswalker** [comment]
All of these comments are on a commit that's meant to be merged on #695. Please check again once that commit is on master.

**Pull request merged**